### PR TITLE
Update ansible packages, enable py3.13

### PR DESCRIPTION
**Pull request by idealseal**
---

Please check all the boxes that apply:

- [x] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [x] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [x] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [x] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

**Comment by thesamesam** [comment]
The ansible-molecule commit changes the doc stuff -- why? The commit message doesn't comment on it.

File: app-admin/ansible-core/ansible-core-2.18.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+	EGIT_REPO_URI="https://github.com/ansible/ansible.git"
+	EGIT_BRANCH="devel"
+else
+	inherit pypi
+	KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~riscv ~x86 ~x64-macos"
+fi
+
+LICENSE="GPL-3"
+SLOT="0"
+RESTRICT="test"
```
[the remark this answers is not in the document]
Comment (idealseal) [reply app-admin/ansible-core/ansible-core-2.18.1.ebuild:29]: I copied that from the previous ebuild. I added some context. I can't get it to work inside the source directory sadly.

File: app-admin/ansible-core/ansible-core-2.18.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517="setuptools"
+PYTHON_COMPAT=(python3_{11..13})
```
[the remark this answers is not in the document]
Comment (idealseal) [reply app-admin/ansible-core/ansible-core-2.18.1.ebuild]: Yeah, that was `shfmt` :disappointed:

**Comment by thesamesam** [comment]
Thanks!

**Pull request closed without merging**